CONTRIBUTING.md

```diff
@@ -43,6 +59,10 @@ If you cannot find an existing issue that describes your bug, create a new issue
 
 File a single issue per problem. Do not enumerate multiple bugs or feature requests in the same issue.
 
+Use the latest version of the AL language extension from the [Visual Studio Code Marketplace](https://marketplace.visualstudio.com/items?itemName=ms-dynamics-smb.al) or shipped as part of the [AL Developer Preview builds](README.md#al-developer-preview-builds) for Dynamics 365 Business Central.
+
+Switch off all extensions except the AL Language extension.
```

Comment:
The VS Code term is disable.

Disable all extensions installed in Visual Studio Code except the AL Language extension.